src/helpers/message.ts

```diff
@@ -56,8 +57,11 @@ export function buildUnknownTokenMsg(order: OrderDto): string | null {
 
   if (!sellToken.known || !buyToken.known) {
     return (
-      '"Maybe" means one or more tokens claim to be called as shown, ' +
-      "but it's not currently part of the list of [known tokens](https://github.com/gnosis/dex-js/blob/master/src/tokenList.json). " +
+      '"Maybe" means one or more tokens claim to be called as shown, but it\'s not currently part of the list of ' +
+      '[known tokens TCR (getTokens with listId = ' +
+      TCR_LIST_ID +
+      ')]' +
+      `(https://etherscan.io/address/${TCR_CONTRACT_ADDRESS}#readContract). ` +
```

Comment:
Should this account for rinkeby, as in `https://rinkeby.etherscan.io/...` or do we not care particularly?

Reply:
Well, I was doing that, but then realized the telegram bot doesn't have the notion of network, I guess?
I thought about an env var but that sounds bad. Maybe getting the networkId from web3?

In the end I went for:
> or do we not care particularly?

Because, well, only we are supposed to see the rinkeby version, so where it matters (prod bot) it'll be correct.

Reply:
I'm fine leaving it like this too :) 

getting the network is easy, just query the node. But just thinking if it's worth it. If you want, go for it, otherwise don't 😌

Reply:
Alright alright, done fd1889b5784b44c82089c44ce2f6eff911eb3995